src/components/va-alert/va-alert.css

```diff
@@ -0,0 +1,137 @@
+@import '../../mixins/accessibility.css';
+
+:host {
```

Comment:
A big chunk of the line count is coming from the classes in here for styling the different statuses.